libsoftwarecontainer/include/gateway.h

```diff
@@ -154,11 +154,13 @@ class Gateway
     static constexpr const char *ENABLED_FIELD = "enabled";
     static constexpr const char *XDG_RUNTIME_DIR_VARIABLE_NAME = "XDG_RUNTIME_DIR";
```

Comment:
While at it, could you also see if the defines make sense to have in the base class?